src/ert/parsing/config_errors.py

```diff
@@ -40,36 +42,25 @@ def _get_error_message(cls, info: ErrorInfo) -> str:
             else info.message
         )
 
-    def get_error_messages(self) -> List[str]:
-        return [self._get_error_message(error) for error in self.errors]
-
     def get_cli_message(self) -> str:
+        return "\n".join(self.get_error_messages())
+
+    def get_error_messages(self) -> List[str]:
         by_filename = defaultdict(list)
         for error in self.errors:
             by_filename[error.filename].append(error)
 
-        def indent_only_first_line(message: str, prefix="  * "):
-            lines = message.splitlines()
-            blank_prefix = " " * len(prefix)
-
-            return "\n".join(
-                [prefix + lines[0], *[(blank_prefix + line) for line in lines[1:]]]
-            )
-
-        result = ";".join(
-            [
-                f"Parsing config file `{filename}` resulted in the errors: \n"
-                + "\n".join(
-                    [
-                        indent_only_first_line(err.message_with_location, "  * ")
-                        for err in info_list
-                    ]
+        nice_messages = []
```

Comment:
Nice code cleanup :+1: would you consider renaming this variable to just `messages`? I know it is not as nice as `nice_messages` (pun intended) but a bit shorter  :smile:.

Reply:
Done!